### PR TITLE
feat(perf): Add min and max exclusive time filters to event examples

### PR DESCRIPTION
**Pull request by udameli**
This PR adds an ability to pass `min_exclusive_time` and `max_exclusive_time` to the span event example query, so that returned events contain spans with exclusive time satisfying the criteria. The reason we need to specify min and max is because we want to enable users to see specific example events that correspond to the selected population when users zoom in the histogram in the span details page.

Edit: `min_exclusive_time` and `max_exclusive_time` can be provided independently, so I've added more cases when one of them is not defined.

Fixes PERF-1518

### CODE REVIEW

File: src/sentry/api/endpoints/organization_events_spans_performance.py [hunk 1]
```diff
@@ -523,10 +548,36 @@ def query_suspect_span_groups(
...
         group = span.group
 
+        op_group_match_condition = Function(
+            "and",
+            [
+                Function("equals", [Identifier("x"), op]),
+                Function("equals", [Identifier("y"), group]),
+            ],
+        )
+        condition = op_group_match_condition
```
Comment (dashed) [inline src/sentry/api/endpoints/organization_events_spans_performance.py:569]:
```suggestion
        condition = Function(
            "and",
            [
                Function("equals", [Identifier("x"), op]),
                Function("equals", [Identifier("y"), group]),
            ],
        )
```

**Review by dashed — APPROVED** [review]

File: src/sentry/api/endpoints/organization_events_spans_performance.py [hunk 2]
```diff
@@ -666,6 +732,33 @@ def get_example_transaction(
         if span["op"] == span_op and int(span["hash"], 16) == span_group_id
     ]
 
+    if min_exclusive_time is not None:
```
Comment (Zylphrex) [inline src/sentry/api/endpoints/organization_events_spans_performance.py:735]: something analogous to the above should be easier to read. iterating the list of spans up to 3 times isn't that bad in terms of performance and keeps the code much cleaner

**Review by Zylphrex — APPROVED** [review]

File: src/sentry/api/endpoints/organization_events_spans_performance.py (hunk 2 above)
Comment (udameli) [reply src/sentry/api/endpoints/organization_events_spans_performance.py:723]: done!

**Comment by udameli** [comment]
accidentally pushed frontend changes, will clean up

**Pull request merged**